ember_debug/object-inspector.js

```diff
@@ -1037,6 +1044,9 @@ function getDebugInfo(object) {
   let debugInfo = null;
   let objectDebugInfo = get(object, '_debugInfo');
   if (objectDebugInfo && typeof objectDebugInfo === 'function') {
+    if (object instanceof Ember.ObjectProxy && object.content) {
+      object = object.content;
```

Comment:
I wonder if maybe we should use `Ember.get`, wouldn't that resolve proxy values as well as normal objects?

Reply:
but the new ember data debugInfo  does not do it...
always fails here:
https://github.com/emberjs/data/blob/master/packages/model/addon/-private/model.js#L1186

Reply:
@patricklx can you elaborate on that? Perhaps @runspired can help?

Reply:
> but the new ember data debugInfo 

I wouldn't call this "new", its been around a long time and it's something we'd love to delete :P 

Reaching through the proxy for the content seems wise here.

Reply:
mmm, right...
but I think it did not happen before...  maybe because im using class extends Model {...} instead of Model.extend(...)

Reply:
no, same issue, so might have never worked with belongTo relations :)